scripts/cleanupTest.js

```diff
@@ -3,16 +3,16 @@ const rimraf = require('rimraf');
 const { join } = require('path');
 const collectTestFolders = require('./utils');
 
-const outputDirectories = ['bin', 'binary', 'dist', 'test', 'test-assets', 'test-plugin', 'test-loader', 'stats.json'];
+const outputDirectories = ['bin', 'binary', 'dist', 'lib', 'test', 'test-assets', 'test-plugin', 'test-loader', 'stats.json'];
```

Comment:
which test is creating lib?

Reply:
`loader-error-test` but it's creating it at the root `/lib` but I think the script cleans only folders created inside the test folder `/test/**`?

Reply:
yep base dir is test, maybe create the output file in the test directory itself and name it `dist` or something

Reply:
updated 👍🏻